UnitTest.Commands/html/partials/_report_summary.liquid

```diff
@@ -86,36 +86,36 @@
                 </tr>
                 <tr>
                   <td style="width:300px;">Application Types</td>
-                  <td style="">{% for item in apptypes -%}{{item}} {% endfor -%}</td>
+                  <td style="">{% for item in apptypes -%}{{item}}, {% endfor -%}</td>
```

Comment:
These will have a comma appended to it. I think you can use the 'join' template tag:

```
{{ apptypes | join:", " }}
```